templates/frontend/pages/indexJournal.tpl

```diff
@@ -23,26 +23,59 @@
 <main class="page_index_journal">
 	<div class="container-fluid container-page">
 
+		{* Announcements *}
+		{if $announcements}
+			<section class="announcements">
+				<h2>{translate key="announcement.announcements"}</h2>
+				<div class="row">
+					{foreach from=$announcements item=announcement}
+						<article class="col-md-4 announcement">
+							<p class="announcement__date">{$announcement->getDatePosted()|date_format:$dateFormatShort|escape}</p>
+							<h3 class="announcement__title">
+								<a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()|escape}">
+									{$announcement->getLocalizedTitle()|escape}
+								</a>
+							</h3>
+						</article>
+					{/foreach}
+				</div>
+			</section>
+		{/if}
+
 		{call_hook name="Templates::Index::journal"}
 
 		{* Latest issue *}
 		{if $issue}
-			<div class="current_issue">
-				{strip}
-					<span class="current_issue_label">{translate key="journal.currentIssue"}</span>
-					{if $issueIdentificationString}
-						<h1 class="current_issue_title">{$issueIdentificationString|escape}</h1>
+			<section class="current_issue">
+				<header>
+					{strip}
+						<p class="current_issue_label">{translate key="journal.currentIssue"}</p>
+						{if $issueIdentificationString}
+							<h2 class="current_issue_title">{$issueIdentificationString|escape}</h2>
+						{/if}
+					{/strip}
```

Comment:
So, the potential risk here is that there is no identification string and as a result there is no `<h2>`. An alternative approach that you can use is to consider the current issue and the issue title all part of the heading:

```
<h2>
  <span>Current Issue</span>
  <span>Special Issue on Whatever This is Called</span>
</h2>
```

It's good to be cautious about making headings too long, but in this case I think the benefits outweigh the drawbacks.

Reply:
It’s possible for issues to also not have any volume numbers, nor issue numbers?

Reply:
I _think_ so. I think it needs to have at least one of Volume, Issue, Year, Title, but it may be any one of them.